fabric-loot-tables-v1/src/main/java/net/fabricmc/fabric/api/loot/v1/FabricLootPool.java

```diff
@@ -29,6 +29,10 @@
  * Fabric API is present. Contains accessors for various fields.
  */
 public interface FabricLootPool {
+	static FabricLootPool from(LootPool vanilla) {
```

Comment:
`of`? (Same for FabricLootSupplier)

Reply:
IMO `from` means they may be the same thing while `of` means the argument is just a component